### DISCUSSION TITLE
Use gfm with rehype/remark plugins

### DISCUSSION THREAD

**Discussion by s7tya** [remarkjs/remark#718] [Q&A] [answered]
]I'm using react markdown with syntax/math(example on README)

I'd like to use ReactMarkdown with GFM, but setting like:

```
 <ReactMarkdown
                plugins={[gfm, breaks]}
                children={body}
                components={{ ...  }}
                remarkPlugins={[remarkMath]}
                rehypePlugins={[rehypeKatex]}
              />
```

then, only remark/rehype plugins work and gfm doesnt work.... how do i fix this?

**Comment by ChristianMurphy** [comment, marked as answer]
Assuming this is react-markdown 6.
Move `gfm` and `breaks` from `plugins` to `remarkPlugins`

**Reply by s7tya** [reply]
Thanks! Great.
